test/plugins/test_case_time_log_test.py

```diff
@@ -73,3 +77,22 @@ def test_json_reporter_reports(self):
                 json.loads(self.reporter.log_file.getvalue()),
                 json.loads(output_str),
             )
+
+    def test_test_case_results_reports(self):
```

Comment:
I'd delete the unit test since this entirely covers the behaviour without fragile mocking